Hackney.Shared.HousingSearch/Domain/Tenure/Tenure.cs

```diff
@@ -29,6 +29,7 @@ public Tenure()
             HouseholdMembers = houseHoldMembers != null && houseHoldMembers.Any() ? houseHoldMembers.Select(HouseholdMember.Create).ToList() : new List<HouseholdMember>();
             TenureType = TenureType.Create(tenureType);
             TenuredAsset = TenuredAsset.Create(tenuredAsset);
+            TemporaryAccommodationInfo = TemporaryAccommodationInfo.Create(queryableTemporaryAccommodationInfo);
```

Comment:
On the frontend we were assuming this would match the Tenure API name `TempAccommodationInfo` as in https://github.com/LBHackney-IT/tenure-shared/blob/ad01b70dcc8b0acf1f1d34989a9bb8c44c72fa0e/Hackney.Shared.Tenure/Boundary/Response/TenureResponseObject.cs#L35. I think it'd be best to keep the name consistent, whatever we choose

Reply:
Yeah, the field name for the Tenure API is `TempAccommodationInfo`.
Would need to take a look at the housing listener logic to see whether different names will cause any issues _(it could if they're using that EntityUpdater contraption)_.

Reply:
@Duslerke I believe you are referring to the housing search listener. It doesn't use the EntityUpdater, but uses a simple factory instead https://github.com/LBHackney-IT/housing-search-listener/blob/129b12a0f4e12856f0447dc53e8fc9afb9dffe16/HousingSearchListener/V1/Factories/ESEntityFactory.cs#L48. For consistency it would probably be best to use the same name in all components, including the tenure search index object. At this stage we should probably follow the Tenure API name and update it here and in the search listener.

Reply:
Renamed in [75d152c](https://github.com/LBHackney-IT/housing-search-shared/pull/73/commits/75d152c7b7460261401b9912c94e4f3be3bc9a9e).